### PR TITLE
Trim the code.

### PR DESCRIPTION
**Pull request by Yuseon-Han**
- It isn't need to be Map

### CODE REVIEW

File: nugu-core/src/main/java/com/skt/nugu/sdk/core/directivesequencer/DirectiveRouter.kt [hunk 1]
```diff
@@ -15,28 +15,26 @@
...
 import com.skt.nugu.sdk.core.utils.Logger
-import java.util.concurrent.ConcurrentHashMap
 
 class DirectiveRouter {
     companion object {
         private const val TAG = "DirectiveRouter"
     }
 
-    private val handlerAndConfigurationMap = ConcurrentHashMap<DirectiveHandler, Map<NamespaceAndName, BlockingPolicy>>()
+    private val handlers = HashSet<DirectiveHandler>()
```
Comment (shshshl09) [inline nugu-core/src/main/java/com/skt/nugu/sdk/core/directivesequencer/DirectiveRouter.kt:29]: check thread-safety

Comment (Yuseon-Han) [reply nugu-core/src/main/java/com/skt/nugu/sdk/core/directivesequencer/DirectiveRouter.kt:29]: I got this :)

File: nugu-core/src/main/java/com/skt/nugu/sdk/core/directivesequencer/DirectiveRouter.kt [hunk 2]
```diff
@@ -15,28 +15,28 @@
...
     companion object {
         private const val TAG = "DirectiveRouter"
     }
 
-    private val handlerAndConfigurationMap = ConcurrentHashMap<DirectiveHandler, Map<NamespaceAndName, BlockingPolicy>>()
+    private val handlers = Collections.synchronizedSet(HashSet<DirectiveHandler>())
 
     fun addDirectiveHandler(handler: DirectiveHandler): Boolean {
-        handlerAndConfigurationMap[handler] = handler.getConfiguration()
+        handlers.add(handler)
```
Comment (tommy5oo) [inline nugu-core/src/main/java/com/skt/nugu/sdk/core/directivesequencer/DirectiveRouter.kt:34]: Check the return result value.
like this:
```kotlin
return handlers.add(handler)
return handlers. remove(handler)
```

Comment (Yuseon-Han) [reply nugu-core/src/main/java/com/skt/nugu/sdk/core/directivesequencer/DirectiveRouter.kt:34]: The original code dosen't return a result of 'add'
So I'm understanding that the return value just means 'done' not 'success'.
Any history about this?

Comment (Yuseon-Han) [reply nugu-core/src/main/java/com/skt/nugu/sdk/core/directivesequencer/DirectiveRouter.kt:34]: I found no-one  use this return value. I removed return type.

**Review by shshshl09 — APPROVED** [review]

File: nugu-core/src/main/java/com/skt/nugu/sdk/core/directivesequencer/DirectiveRouter.kt (hunk 2 above)
Comment (tommy5oo) [reply nugu-core/src/main/java/com/skt/nugu/sdk/core/directivesequencer/DirectiveRouter.kt:34]: Yes! Thanks!

**Review by tommy5oo — APPROVED** [review]

**Pull request merged**